tools/poretools/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: poretools
```

Comment:
@shiltemann Gebruik je nanpore data voor een EMC project? Zoniet, is dit niet meer op zijn plaats in IUC?